app/components/aptible-ability/component.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+import can from '../../utils/can';
+
+export default Ember.Component.extend({
+  hasAbility: false, // cannot use `isVisible` with `tagName: ''`
```

Comment:
This usage of a component feels awkward b/c you're basically writing an `if` helper, but I think your solution here is idiomatic. +1.

To do it differently (more efficiently or just less weird feeling) we would need to muck with the `render` method or write a raw HTMLBars helper.